### PR TITLE
added xPath & methods for AddChallengePage & EditChallengePage

### PR DESCRIPTION
**Pull request by janestadnytska**

### CODE REVIEW

**Review by TetianaFilatova — APPROVED** [review]

File: src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java [hunk 1]
```diff
@@ -1,10 +1,104 @@
 package org.ssu.edu.teachua.ui.pages.challenges;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
 import org.ssu.edu.teachua.ui.base.BasePage;
 
 public class AddChallengePage extends BasePage {
+    @FindBy(how = How.XPATH, using = ".//input[@id='sortNumber']")
```
Comment (Maksym637) [inline src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java:10]: This XPATH should not start with `.` (Only in classes inherited by BaseComponent)

**Review by Maksym637 — APPROVED** [review]

File: src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java [hunk 2]
```diff
@@ -1,10 +1,104 @@
...
+
+
+    public AddChallengePage addPhoto(String imagePath) {
+        getUploadPhoto().sendKeys(imagePath);
+        return this;
+    }
+
+    public AddChallengePage clickSave() {
+        getSaveButton().click();
+        return new AddChallengePage(driver);
```
Comment (lhalam) [inline src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java:93]: return this

File: src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java [hunk 3]
```diff
@@ -1,10 +1,104 @@
...
+    }
+
+    public AddChallengePage goToChallenges() {
+        getGoToChallengesBtn().click();
+        return new AddChallengePage(driver);
```
Comment (lhalam) [inline src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java:98]: return ChallengesPage

Comment (janestadnytska) [reply src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java:98]: done

**Comment by Maksym637** [comment]
Nice job !

**Pull request merged**